spec/system/accessibility_page_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.describe "accessibility page", type: :system do
+  let(:contact_details) do
+    {
+      default: {
+        email_address: "planning@default.gov.uk",
+        phone_number: "01234123123",
+        postal_address: "Planning, 123 High Street, Big City, AB3 4EF",
+      },
+    }.deep_stringify_keys
+  end
+
+  before do
+    ENV["API_BEARER"] = "123"
+    ENV["PROTOCOL"] = "https"
```

Comment:
Out of scope for this PR but reminds me that we should extract this horrible thing as it's in every spec

Reply:
What do you think about using Climate Control? https://github.com/thoughtbot/climate_control